docroot/sites/archaeology/settings.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+require DRUPAL_ROOT . "/sites/default/settings.php";
+
+if (file_exists('/var/www/site-php')) {
```

Comment:
This seems like a manual process that could be made more dynamic. I suppose you could scan that directory for settings.inc files and include them all. This is good for now.